example/config/README.md

```diff
@@ -0,0 +1,249 @@
+
+- `user_metadata`: `'severe_violations'`
+
+### Model Grouping (optional)
+Model groups are a way of partitioning trained models in a way that makes for easier analysis.
+
+`model_group_keys` defines a list of training matrix metadata and classifier keys that should be considered when creating a model group.
+
+There is an extensive default configuration, which is aimed at producing groups whose constituent models are equivalent to each other in all ways except for when they were trained. This makes the analysis of model stability easier.
+
+To accomplish this, the following default keys are used: `class_path`, `parameters`, `feature_names`, `feature_groups`, `cohort_name`, `state`, `label_name`, `label_timespan`, `as_of_date_frequency`, `max_training_history`
+
+If you want to override this list, you can supply a `model_group_keys` value. All of the defaults are available, along with some other temporal information that could be useful for more specialized analyses:
+
+`first_as_of_time`, `last_as_of_time`, `matrix_info_end_time`, `as_of_times`, `feature_start_time`
+
+You can also use any pieces of user_metadata that you included in this experiment definition, as they will be present in the matrix metadata. 
+- `model_group_keys`: [`feature_groups`, `label_definition`]
+
+### Grid Configuration
+The classifier/hyperparameter combinations that should be trained
+
+Each top-level key should be a class name, importable from triage. sklearn is available, and if you have another classifier package you would like available, contribute it to requirement/main.txt
+
+- `grid_config`: Each lower-level key is a hyperparameter name for the given classifier, and each value is a list of potential values. All possible combinations of classifiers and hyperparameters are trained. Please check out the [grid_config session](https://github.com/dssg/triage/blob/6cb43f9cca032a980cbc25a9501e9559135fd04d/example/config/experiment.yaml#L276) in `experiment.yaml` as for a detailed example.
+
+
+### Prediction
+How predictions are computed for train and test matrices?
```

Comment:
Might be worth noting here that this is used only for _stored_ predictions and only affect postmodeling analysis (not model scoring), so if you are not storing predictions, this will not affect anything.